extract_data.py

```diff
@@ -164,11 +117,11 @@ def normalize_brands(coll_dict):
 # mounting psu
```

Comment:
[formal] please refactor **everywhere** as follows (line up indentation)
```suggestion
    # mounting psu
```